src/aof.c

```diff
@@ -687,6 +688,20 @@ void aofOpenIfNeededOnServerStart(void) {
         exit(1);
     }
 
+    /* If we start with an empty dataset and ‘aof_use_rdb_preamble’ is set to yes, we 
+     * will force create a BASE (rdb format) file. */
+    if (server.aof_use_rdb_preamble == 1 &&
```

Comment:
BTW, if we remove the `server.aof_use_rdb_preamble == 1` check, so that we can always have a BASE file, @oranagra WDYT?

Reply:
Done in this way.